examples/graph/wgan-graphs.py

```diff
@@ -0,0 +1,545 @@
+"""
+Title: WGAN-GP with R-GCN for the generation of small molecular graphs
+Author: [akensert](https://github.com/akensert)
+Date created: 2021/06/27
+Last modified: 2021/06/27
+Description: Complete implementation of WGAN-GP with R-GCN to generate novel molecules.
+"""
+
+"""
+## References
+
+The implementation in this tutorial is based on/inspired by the [MolGAN
+paper](https://arxiv.org/abs/1805.11973) and DeepChem's [Basic
+MolGAN](https://deepchem.readthedocs.io/en/latest/api_reference/models.html#basicmolganmod
+el).
+
+## Further reading
+Recent implementations of generative models for molecular graphs also include
+[Mol-CycleGAN](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0404-1),
+[GraphVAE](https://arxiv.org/abs/1802.03480) and
+[JT-VAE](https://arxiv.org/abs/1802.04364). For more information on generative
+adverserial networks, see [GAN](https://arxiv.org/abs/1406.2661),
+[WGAN](https://arxiv.org/abs/1701.07875) and [WGAN-GP](https://arxiv.org/abs/1704.00028).
+
+"""
+
+"""
+## Setup
+
+### Install RDKit
+
+[RDKit](https://www.rdkit.org/) is a collection of cheminformatics and machine-learning
+software written in C++ and Python. In this tutorial, RDKit is used to conviently and
+efficiently transform
+[SMILES](https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system) to
+molecule objects, and then from those obtain sets of atoms and bonds.
+
+Notice, RDKit is commonly installed via [Conda](https://www.rdkit.org/docs/Install.html).
+However, thanks to [kuelumbus'](https://github.com/kuelumbus/)
+[rdkit_platform_wheels](https://github.com/kuelumbus/rdkit_platform_wheels), rdkit
+can now (for the sake of this tutorial) be installed easily via pip.
+"""
+
+"""shell
```

Comment:
Rather than including this block, list the required dependencies and show a code snippet of the pip commands. Readers should be able to choose to install them.